build/lily/docker_init.sh

```diff
@@ -22,6 +22,6 @@ else
   lily init
```

Comment:
I can't comment on the line, but above on line 11, could we add an `elif` with something like:
```bash
elif [[ ! -z "${LILY_DOCKER_INIT_IMPORT_SNAPSHOT_PATH}" ]]; then
    echo "Importing snapshot from ${LILY_DOCKER_INIT_IMPORT_SNAPSHOT_PATH}"
    lily init --import-snapshot=${LILY_DOCKER_INIT_IMPORT_SNAPSHOT_PATH}
```
then we could optionally specify locally mounted snapshots in the compose file: https://github.com/filecoin-project/lily/pull/1044/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R86

Reply:
I made the envvar get a default assignment if found to be empty.